.dumi/scripts/mirror-modal.js

```
@@ -17,118 +18,130 @@
-    font-size: 16px;
-    font-weight: 500;
-    align-self: flex-start;
-    margin-bottom: 8px;
-  }
-
-  .mirror-modal-content {
-    font-size: 14px;
-    align-self: flex-start;
-    margin-bottom: 24px;
+    let success = false;
+    fetch(`${window.location.href}?t=${Date.now()}`).then(() => (success = true));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)